apps/nextjs-website/src/lib/api.ts

```diff
@@ -95,10 +100,31 @@ export async function getQuickStartGuide(productSlug?: string) {
       ({ product }) => product.slug === productSlug
     )
   );
-  return manageUndefinedAndAddProduct(props);
+  return manageUndefinedAndAddProducts(props);
+}
+
+export async function getStrapiTutorial(
+  productSlug?: string,
+  productTutorialPage?: ReadonlyArray<string>
+) {
+  const tutorialSubPath = productTutorialPage?.join('/');
+  const tutorialPath = `/${productSlug}/tutorials/${tutorialSubPath}`;
+
+  const tutorialsFromCMS = await getTutorialsProps();
```

Comment:
The function name is `getStrapiTutorial`, why this constant name refers to `CMS`?